### PR TITLE
[Story] Create About Page

### PR DESCRIPTION
**Pull request by ranjabi**
## Story/Task
Create About Page

## Summary
- _Add about page_
- _Add about detail components_
- _Add oskm logo svg_
- _Fix lint script_

## Screenshot
image
image
image
image

## Test Plan
- [x] _Go /about_
- [x] _Click "see more" and "go back" button to navigate the detail content._

Note:
Still waiting for the real content, currently using the dummy content.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
         "build": "yarn lint && tsc && vite build",
         "preview": "vite preview",
         "prepare": "husky install",
-        "lint": "yarn eslint ./src/** --fix && yarn prettier --write .",
-        "lint:auto": "yarn eslint ./src/** --fix --no-color && yarn prettier --write --no-color ."
+        "lint": "yarn eslint --fix --ext .js,.jsx,.ts,.tsx src/ && yarn prettier --write .",
+        "lint:auto": "yarn eslint --fix --no-color --ext .js,.jsx,.ts,.tsx src/ && yarn prettier --write --no-color ."
```
Comment (IloveNooodles) [inline package.json:12]: can we exclude only the /src/font

File: src/pages/About/About.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import oskmLogo from '../../resource/oskm-logo.svg';
+import AboutDetail from './AboutDetail';
+import { useToggle } from '../../hooks/useToggle';
+
+const About = () => {
+    const [isOn, toggle] = useToggle();
+
```
Comment (IloveNooodles) [inline src/pages/About/About.tsx:7]: could you put the content inside of a map nad put the type in the type.ts to simulate the real content

File: src/pages/About/AboutDetail.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import oskmLogo from '../../resource/oskm-logo.svg';
+import { AboutProps } from '../../types/interface';
+
+const AboutDetail: React.FC<AboutProps> = ({ toggle }) => (
```
Comment (IloveNooodles) [inline src/pages/About/AboutDetail.tsx:5]: same for above

**Review by IloveNooodles — CHANGES_REQUESTED** [review]
could you rename the resource to the asset folder

File: package.json (hunk 1 above)
Comment (ranjabi) [reply package.json]: src/font is excluded in the new script

**Pull request merged**